jira.py

```diff
@@ -32,6 +38,82 @@ def __init__(self):
     # API: Filters
     def filters(self):
         query = JQL_QUERY + FILTER_ID_ALL_REQUESTS_2022 + '&fields=' \
-                + DEFAULT_COLUMNS + STORY_POINTS + FIREFOX_RELEASE_TRAIN \
+                + DEFAULT_COLUMNS + ',' + STORY_POINTS + ',' \
+                + FIREFOX_RELEASE_TRAIN + ',' \
                 + ENGINEERING_TEAM + '&' + MAX_RESULT
+
         return self.client.get_search(query)
+
+
+class JiraClient(Jira):
+    def __init__(self):
+        super().__init__()
+        self.db = DatabaseJira()
+
+    def jira_qa_requests(self):
+        payload = self.filters()
+
+        data_frame = self.db.report_jira_qa_requests_payload(payload)
+        print(data_frame)
+
+        self.db.report_jira_qa_requests_insert(data_frame)
+
+
+class DatabaseJira(Database):
+
+    def __init__(self):
+        super().__init__()
+        self.db = Database()
+
+    def report_jira_qa_requests_payload(self, payload):
+        # Normalize the JSON data
+        df = pd.json_normalize(payload, sep='_')
+
+        # Check if 'jira_assignee_username' exists
+        # if not use 'alternative_assignee_emailAddress'
+        if 'fields_assignee_emailAddress' not in df.columns:
```

Comment:
We need this because if issue does not have an assignee, then the fileds related, like assignee name or email do not exist